analysed-packages/tcpdump/version-4.99.1/tcpdump-4.99.1-SPDX2TV.spdx

```diff
@@ -0,0 +1,19385 @@
+PackageChecksum: SHA256: 79b36985fb2703146618d87c4acde3e068b91c553fb93f021a337f175fd10ebe
+PackageChecksum: MD5: 929a255c71a9933608bd7c31927760f7
+PackageLicenseConcluded: NOASSERTION
+PackageLicenseDeclared: LicenseRef-BSD-3-Clause
+PackageLicenseComments: <text> licenseInfoInFile determined by Scanners:
+ - nomos ("4.4.0".903df0)
+ - monk ("4.4.0".903df0)
+ - ojo ("4.4.0".903df0) </text>
+PackageLicenseInfoFromFiles: NOASSERTION
+PackageCopyrightText: NOASSERTION
+
+Relationship: SPDXRef-DOCUMENT DESCRIBES SPDXRef-upload643
+
+
+##--------------------------
+## File Information
+##--------------------------
+
+##File
+
+FileName: tcpdump-4.99.1.tar.gz/tcpdump-4.99.1.tar/tcpdump-4.99.1/print-icmp6.c
+SPDXID: SPDXRef-item1161891
+FileChecksum: SHA1: 27a711901da80c649162a001afa87b3b5c6b0251
+FileChecksum: SHA256: e7e7332b57a879ae8955e485a964187a2f1796f2fe664d8b6caad0285a1f0398
+FileChecksum: MD5: c61c7b08ade0de294fcdeea7786b8ff0
+LicenseConcluded: LicenseRef-BSD-3-Clause-b14dce71205018b7049d6525edb666d8 AND LicenseRef-BSD-4-Clause-Shortened-e96ddab5f8fffe47de17b5812d71176a
+LicenseInfoInFile: LicenseRef-BSD-4-Clause-Shortened
+LicenseInfoInFile: LicenseRef-BSD-3-Clause-HP
+LicenseInfoInFile: LicenseRef-BSD-4-Clause-UC
+FileCopyrightText: <text> Copyright (c) 1988, 1989, 1990, 1991, 1993, 1994 The Regents of the University of California. All rights reserved.
+Copyright (C) 1995, 1996, 1997, and 1998 WIDE Project. All rights reserved. </text>
+
+##File
+
+FileName: tcpdump-4.99.1.tar.gz/tcpdump-4.99.1.tar/tcpdump-4.99.1/print-snmp.c
+SPDXID: SPDXRef-item1161896
+FileChecksum: SHA1: 96404dabd57c3040cf4722028797994b12dd3fad
+FileChecksum: SHA256: b504184591baba7be5b20aa89b2228279fb5e09eb1bb1df161778e0706ffbc3c
+FileChecksum: MD5: 0a76c2b2c3867f234ad568860b76358b
+LicenseConcluded: LicenseRef-BSD-2-Clause-55ee54facd173f8169b9a4625f2f79ec AND LicenseRef-Permission-Notice-3a910945594b4de572e0943feee55bcb
+LicenseInfoInFile: LicenseRef-Less
+LicenseInfoInFile: LicenseRef-BSD-2-Clause
+FileCopyrightText: <text> Copyright (c) 1990, 1991, 1993, 1994, 1995, 1996, 1997 John Robert LoVerso. All rights reserved.
+Copyright (c) 1999.
+Copyright (c) 1990, 1991, 1993, 1994, 1995, 1996, 1997 This software was produced under a U.S. Government contract W-7405-ENG-36) by Los Alamos National Laboratory, which is operated by the	University of California for the U.S. Department of Energy.  </text>
+
+##File
+
+FileName: tcpdump-4.99.1.tar.gz/tcpdump-4.99.1.tar/tcpdump-4.99.1/print-enc.c
+SPDXID: SPDXRef-item1163239
+FileChecksum: SHA1: 1424350f9cb4eeef42da0b897747491b20e35b1c
+FileChecksum: SHA256: c5a8d6de936589748a181111b514bd4393ce7ec647df1fc15941ffacf0b66bf8
+FileChecksum: MD5: b261dff3f15cf19a89b8d3443ef6bd2b
+LicenseConcluded: LicenseRef-BSD-4-Clause-Shortened-e96ddab5f8fffe47de17b5812d71176a AND LicenseRef-Permission-Notice-3a2bf8a18b88925f22a9130d4cf995f0 AND LicenseRef-warranty-disclaimer-88394e5be8e0e24e8312d81013827b8f
```

Comment:
the LicenseRef-Permission-Notice-3a2bf8a18b88925f22a9130d4cf995f0 AND LicenseRef-warranty-disclaimer-88394e5be8e0e24e8312d81013827b8f is a dual license statement:
 Permission to use, copy, and modify this software with or without fee
 is hereby granted, provided that this entire notice is included in
 all copies of any software which is or includes a copy or
 modification of this software.
 You may use this code under the GNU public license if you so wish. Please
 contribute changes back to the authors under this freer than GPL license
 so that we may further the use of strong encryption without limitations to
 all.

 THIS SOFTWARE IS BEING PROVIDED "AS IS", WITHOUT ANY EXPRESS OR
 IMPLIED WARRANTY. IN PARTICULAR, NONE OF THE AUTHORS MAKES ANY
 REPRESENTATION OR WARRANTY OF ANY KIND CONCERNING THE
 MERCHANTABILITY OF THIS SOFTWARE OR ITS FITNESS FOR ANY PARTICULAR
 PURPOSE.
The overall license is 
LicenseRef-BSD-4-Clause-Shortened-e96ddab5f8fffe47de17b5812d71176a AND (GPL-1.0-or-later OR Permission-Notice)